single_tech_samples/datafactory/devops/azure-pipelines-cd-release.yml

```diff
@@ -33,13 +33,6 @@ stages:
         parameters:
           environmentName: 'STG'
           serviceConnection: 'mdwdo-adf-serviceconnection-stg'
-
-  - stage: integration_e2e_test
-    displayName: 'Integration E2E Test'
-    variables:
-      - group: mdwdo-adf-release-stg
-      - group: mdwdo-adf-release-secrets-stg
-    jobs:
       - template: templates/jobs/integration-test-adf-job.yml
```

Comment:
Shall we add _dependsOn_ and _condition_ here to make jobs run in sequence? Or you've tested locally and this configures work as expected? Thanks for your confirmation! :-)

Reply:
Done